src/murmur/MurmurGRPCImpl.h

```diff
@@ -36,6 +36,7 @@ namespace MurmurRPC {
 class MurmurRPCImpl : public QThread {
 		Q_OBJECT;
 		std::unique_ptr<grpc::Server> m_server;
+		volatile bool m_isRunning;
```

Comment:
I'm curious: why did you use the `m_` prefix over the `b_` one for a boolean flag?